ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -9,6 +9,22 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 
 {-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+-- | Property tests for the mempool.
+--
+-- The mempool collects transactions from down-stream nodes, makes them
+-- available to up-stream nodes, and of course provides the pool of transactions
```

Comment:
Aren't `down` and `up` swapped here?

Reply:
I just copied the text, but it makes sense to me: nodes that submit transactions are down-stream.

Reply:
In other words, I see the nodes submitting transactions as dependent on those to which they submit them.